tests/test-db.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const test = require("tape-async");
+// const Knex = require("knex");
+
+const DB = require("../db/DB");
+const getSha1 = require("../sha1-utils");
+/*
+const knexConfig = require("../db/knexfile");
+
+const knex = Knex(knexConfig["tests"]);
+*/
+
+
+test("getSubscribersByHashes accepts hashes and only returns verified subscribers", async t => {
+  const testHashes = [
+    "firefoxaccount@test.com",
+    "unverifiedemail@test.com",
+    "verifiedemail@test.com",
+  ].map(email => getSha1(email));
+  const subscribers = await DB.getSubscribersByHashes(testHashes);
+  for (const subscriber of subscribers) {
+    t.ok(subscriber.verified);
+  }
```

Comment:
Does this need a `t.end()` to match the test file below?
Similarly, do we need any `t.plan(3)` bits, like we had in some previous iterations?

Reply:
I've tried every combination, and also wrapping the `await` line in `try...catch` too. :(

Reply:
Turns out the solution seems to be calling `knex.destroy()` during test suite teardown to kill the DB pool, or possibly disable connection pooling during <kbd>$ npm run test</kbd>.

Ref: https://knexjs.org/#Installation-pooling